api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -502,6 +502,22 @@ public void setDiagnoses(Set<Diagnosis> diagnoses) {
 		return conditions;
 	}
 
+	/**
+	 * Returns only the non-voided conditions for this encounter. If you want <u>all</u>
+	 * conditions, use {@link #getConditions()}
+	 *
+	 * @return list of non-voided conditions for this encounter
+	 * @see #getConditions()
+	 */
+	public Set<Condition> getActiveConditions() {
```

Comment:
@icrc-loliveira  thanks for the work on this ,,,
i see @mks-d  in the ticket description said " boolean parameter should be introduced on a new getter to allow including voided conditions if desired" something like https://github.com/openmrs/openmrs-core/blob/1c21cb563496bbfeb58133b0ad2ea8b39121783e/api/src/main/java/org/openmrs/Encounter.java#L320

Reply:
Hi @gitcliff,
I have done the changes that you suggested.